### DISCUSSION TITLE
Updating?

### DISCUSSION THREAD

**Discussion by B3astTech** [ruanformigoni/gameimage#29] [General]
This seems like a really awesome project. But I don't see an exact way to easily update the appimages. Programs like wine and yuzu, etc, are always receiving updates that fix games or performance. 

Is there a way to update the software in the appimage without ripping out the data and repackaging it?

Thanks for your time, and I hope this isn't a stupid question 😅

**Comment by ruanformigoni** [comment]
Hi @B3astTech , thanks for the question!  Currently the best you can do is to not package the wine appimage with the application, and point to it after the packaging process. That way you can update the wine appimage and the application stays intact. A better way would be a package format that allows updates, I've worked a bit on this [here](https://github.com/ruanformigoni/arts), it is still a WIP however.
